### ISSUE TITLE
Code should follow PEP8 guidelines

### ISSUE THREAD

**Issue by bhavin192** [miniSat/minisat#38]
Following errors are found when `flake8` is run,
branch: `testing`

```shell
(env) $ pip install flake8
(env) $ flake8 --ignore=E501 mini main_app 
mini/urls.py:18:25: E126 continuation line over-indented for hanging indent
mini/urls.py:20:25: E123 closing bracket does not match indentation of opening bracket's line
main_app/forms.py:2:1: F401 'django.forms.TextInput' imported but unused
main_app/forms.py:3:1: F403 'from .models import *' used; unable to detect undefined names
main_app/forms.py:25:1: E302 expected 2 blank lines, found 1
main_app/forms.py:32:17: F405 'Container_model' may be undefined, or defined from star imports: .models
main_app/forms.py:38:17: F405 'Local_image_model' may be undefined, or defined from star imports: .models
main_app/urls.py:18:2: W292 no newline at end of file
main_app/models.py:2:1: F401 'django.forms' imported but unused
main_app/models.py:24:5: E303 too many blank lines (2)
main_app/models.py:45:5: E303 too many blank lines (2)
main_app/models.py:55:41: E231 missing whitespace after ','
main_app/models.py:65:1: W391 blank line at end of file
main_app/tests.py:1:1: F401 'django.test.TestCase' imported but unused
main_app/admin.py:4:22: E126 continuation line over-indented for hanging indent
main_app/admin.py:10:21: E126 continuation line over-indented for hanging indent
main_app/views.py:1:1: F401 'django.views.generic.TemplateView' imported but unused
main_app/views.py:2:14: E401 multiple imports on one line
main_app/views.py:5:21: E126 continuation line over-indented for hanging indent
main_app/views.py:11:21: E123 closing bracket does not match indentation of opening bracket's line
main_app/views.py:12:1: F401 'django.http.HttpResponse' imported but unused
main_app/views.py:13:26: E126 continuation line over-indented for hanging indent
main_app/views.py:15:26: E123 closing bracket does not match indentation of opening bracket's line
main_app/views.py:17:21: E126 continuation line over-indented for hanging indent
main_app/views.py:21:21: E123 closing bracket does not match indentation of opening bracket's line
main_app/views.py:34:54: E128 continuation line under-indented for visual indent
main_app/views.py:44:17: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:44:19: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:45:23: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:45:25: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:46:26: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:46:28: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:62:25: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:62:27: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:63:16: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:63:18: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:64:17: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:64:19: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:65:22: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:65:24: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:79:112: E231 missing whitespace after ':'
main_app/views.py:79:136: E231 missing whitespace after ':'
main_app/views.py:79:165: E231 missing whitespace after ':'
main_app/views.py:92:20: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:92:22: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:93:24: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:93:26: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:103:20: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:103:22: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:104:18: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:104:20: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:105:30: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:105:32: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:106:27: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:106:29: E251 unexpected spaces around keyword / parameter equals
main_app/views.py:125:74: E231 missing whitespace after ':'
main_app/views.py:125:98: E231 missing whitespace after ':'
main_app/views.py:139:21: E231 missing whitespace after ','
main_app/views.py:147:111: E231 missing whitespace after ':'
main_app/views.py:147:131: E231 missing whitespace after ':'
main_app/views.py:155:53: E226 missing whitespace around arithmetic operator
main_app/views.py:155:57: E226 missing whitespace around arithmetic operator
main_app/views.py:156:78: E231 missing whitespace after ','
main_app/views.py:159:37: W292 no newline at end of file
```

**Comment by khomesh24** [comment]
@CURIOUSER005 @yadneshk  @jadhavganesh please follow the PEP8 guideline and check before pushing the code.

**Comment by bhavin192** [comment]
@khomesh24 is it fixed? if yes by which commit or PR?

**Comment by khomesh24** [comment]
fixed #42
